### DISCUSSION TITLE
(use case) git-bug and usql?

### DISCUSSION THREAD

**Discussion by ghost** [git-bug/git-bug#954] [Q&A] [answered]
Hi.

There is an interesting project in go called [xo/usql](https://github.com/xo/usql) which is for transferring and storing different databases. 

*but... what is usql?*
Universal command-line interface for SQL databases written in go.

*why this doubt or feature? why did you write this here?*
1. I want to know what the community thinks about this new idea
2. That's my last question.
3. I want to know what you all think about the idea of using usql as a backend for git-bug.
4. I've talked about previous ideas of providing an orbit-db database for git-bug, but what I realized is that there would be a need to bridge different databases instead of one database
5. I thought of using the usql project for the backend (which has incredible support for more different databases), which in my opinion would be a great implementation of mine for the usql community and the git-bug community

**Comment by MichaelMure** [comment, marked as answer]
I've probably written that many times already, but nothing prevent implementing the repository interfaces (the base layer in git-bug) with a SQL backend. I suppose that for large scale infrastructure that could be useful, but appart from that I have to say I don't really see the point.
